### PR TITLE
Request to add me back to translator section

### PR DESCRIPTION
**Pull request by ChillTick**
I was removed from the "Translator" section in [this file](https://github.com/Armin2208/Windows-Auto-Night-Mode/blob/master/AutoDarkModeApp/Properties/Resources.nl.resx) in a previous [commit](https://github.com/Armin2208/Windows-Auto-Night-Mode/commit/285535a874477b1fc1681fb5ef2c4169db8f8f72). I did the original translation for this file and understand that @QkeleQ10 made significant changes to the file but I personally don't think it warrants removing my credit entirely and would hereby request for it to be re-added.

### CODE REVIEW

**Comment by QkeleQ10** [comment]
WAIT- did I forget to include you again? Wow I'm so sorry dude, I definitely didn't mean to do so.

Though I completely rewrote the translation, your previous translations helped me out a lot so you should definitely receive credit. I'm sorry once again.

File: AutoDarkModeApp/Properties/Resources.nl.resx [hunk 1]
```diff
@@ -231,7 +231,7 @@ Momenteel geïnstalleerde versie: {0}, nieuwe versie: {1}</value>
     <value>Vink het selectievakje aan om automatische themawisseling in te schakelen</value>
   </data>
   <data name="lblTranslator" xml:space="preserve">
-    <value>Vertaler: Quinten Althues</value>
+    <value>Vertaler: ChillTick en Quinten Althues</value>
```
Comment (QkeleQ10) [inline AutoDarkModeApp/Properties/Resources.nl.resx:234]: Vertalers* for plural

**Comment by Armin2208** [comment]
Im kinda curios why I didn't notice that haha :D Glas everything is fine.

**Comment by ChillTick** [comment]
Glad to hear it was just a simple mistake 🙂. I changed "Vertaler" to "Vertalers", which should make this pull request ready to be merged.

**Pull request merged**